pkg/windows/windows.go

```diff
@@ -613,11 +632,11 @@ func (vm *windows) ensureHostNameAndContainersFeature() error {
 
 // isHostNameChangeNeeded tells if we need to update the host name of the Windows VM
 func (vm *windows) isHostNameChangeNeeded() (bool, error) {
-	out, err := vm.Run("hostname", true)
+	hostName, err := vm.GetHostname()
 	if err != nil {
-		return false, fmt.Errorf("error getting the host name, with stdout %s: %w", out, err)
+		return false, err
```

Comment:
do we need to get rid of this fmt?

Reply:
it is not going away, it just got moved into the new helper function

```go
func (vm *windows) GetHostname() (string, error) {
	hostName, err := vm.Run(GetHostnameFQDNCommand, true)
	if err != nil {
		return "", fmt.Errorf("error getting the host name, with stdout %s: %w", hostName, err)
	}
	return strings.TrimSpace(hostName), nil
}
```